plugin/src/main/java/com/denizenscript/denizen/objects/LocationTag.java

```diff
@@ -4350,6 +4350,40 @@ else if (material.hasModernData() && material.getModernData() instanceof org.buk
                 brushableBlock.setItem(item.getItemStack());
                 brushableBlock.update();
             });
+
+            // <--[tag]
+            // @attribute <LocationTag.waxed>
+            // @returns ElementTag(Boolean)
+            // @mechanism LocationTag.waxed
+            // @group world
+            // @description
+            // Returns whether the location is a sign block that is waxed.
```

Comment:
`If the location is a sign block, returns whether it is waxed (locked to prevent players editing the text).`